cli/script/command-executor.ts

```diff
@@ -823,14 +823,14 @@ function getReactNativeProjectAppVersion(platform: string, projectName: string):
     var missingPatchVersionRegex: RegExp = /^\d+\.\d+$/;
     if (platform === "ios") {
         try {
-            var infoPlistContainingFolder: string = path.join("iOS", projectName);
+            var infoPlistContainingFolder: string = path.join("ios", projectName);
             var infoPlistContents: string = fs.readFileSync(path.join(infoPlistContainingFolder, "Info.plist")).toString();
```

Comment:
This casing is always used. If anything, we have a feature request to enable specifying a different name and/or location, but if the app is using Info.plist, it's always cased this way.